events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -0,0 +1,347 @@
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoDog.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoPlinks.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoPianoPlinks.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoBomb.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoSqueak.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoChatter.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoClang.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoGiggles.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoSplash.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoWaHaHa.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoFall.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoTwinkle.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoPopUp.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoWhistle.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoBubbles.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoHarp.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoKazoo.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoVibration.wav")));
+        headerSounds.push(SoundCache.getSound(Script.resolvePath("assets/sounds/bingoSpiral.wav")));
+        
+        for (var i = 0; i < BINGO_STRING.length; i++) {
+            getRandomSound();
+        }
+    
+    }
+
+    /* STORE USER: Store the user's name and assigned bingo card numbers for future reference*/
+    var SPREADSHEET_URL = Script.require(Script.resolvePath('../secrets/bingoSheetURL.json?0')).sheetURL;
+    var PLAYER_COUNTER_TEXT = "{15d6a1a1-c361-4c8e-8b9a-f4cb4ae2dd83}";
+    var request = Script.require('request').request;
+    var userName = AccountServices.username;
+    var userCardNumbers = [];
+    function createUserInSheet() {
+        var addParamString = encodeURLParams({
+            type: "add",
+            username: userName
+        });
+        request({
+            uri: SPREADSHEET_URL + "?" + addParamString
+        }, function (error, response) {
+            if (error || !response || response === "Not a new user") {
+                return;
+            }
+            var userNumbersToSplit = response.substring(1, response.length - 1);
```

Comment:
same comment as [here](https://github.com/highfidelity/hifi-content/pull/250/files#r248801413).

this is more of a comment on the google script design than this script: make sure your response contains some information about what the response contains (very meta). for example, your response for this MAY look like:
```
{
  "status": "success",
  "data": {
    "userCardNumbers": <the data>
  }
}
```

that way, you can check to make sure that the response is actually a success AND that it actually contains a valid `data.userCardNumbers` array.